docs-src/concepts/what-is-a-list-filter.md

```diff
@@ -33,17 +33,16 @@ A List Filter contains [Search Attribute](/concepts/what-is-a-search-attribute)
   - **ORDER BY**
   - **LIKE**
 
-- Wildcards ('\*', '%', etc.) are supported for string-type List Filter Search Attributes.
-  Use wildcards with the **LIKE** operator to query possible values:
-
-  ```bash
-  // Create a "ProductId" custom Search Attribute of type String
-  tctl admin cluster add-search-attributes --name ProductId --type String
-  // Match values that start with "book"
-  ProductId LIKE "book%"
-  // Match values that contain "favorite"
-  ProductId LIKE "%favorite%"
+  The last two operators, **ORDER BY** and **LIKE**, are only supported with Elasticsearch as a Visibility store.
+
+- To search for a specific token, [set a custom Search Attribute of type `Text`](/app-dev-context/observability#custom-search-attributes) that has the value of the token.
```

Comment:
I think you rather need to explain what token is (in pop up of something) or replace it with more common word "word".

Reply:
I read this sentence few times and I don't get it. Set is set, search is search. First, you set value of Search Attribute of type `Text` to something like "The quick brown fox jumps over the lazy dog" and then search for specific word "fox".